tests/unit/unit1979.c

```diff
@@ -0,0 +1,177 @@
+/***************************************************************************
+ *                                  _   _ ____  _
+ *  Project                     ___| | | |  _ \| |
+ *                             / __| | | | |_) | |
+ *                            | (__| |_| |  _ <| |___
+ *                             \___|\___/|_| \_\_____|
+ *
+ * Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+ *
+ * This software is licensed as described in the file COPYING, which
+ * you should have received as part of this distribution. The terms
+ * are also available at https://curl.se/docs/copyright.html.
+ *
+ * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+ * copies of the Software, and permit persons to whom the Software is
+ * furnished to do so, under the terms of the COPYING file.
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+ * KIND, either express or implied.
+ *
+ * SPDX-License-Identifier: curl
+ *
+ ***************************************************************************/
+ #include "curlcheck.h"
+ #include "http_aws_sigv4.h"
+ #include "dynbuf.h"
+ 
+ 
+ static CURLcode unit_setup(void)
+ {
+   return CURLE_OK;
+ }
+ 
+ static void unit_stop(void)
+ {
+ }
+ 
+ 
+ struct testcase {
+   const char *testname;
+   const bool normalize;
+   const char *url_part;
+   const char *canonical_url;
+ };
+ 
+ static const struct testcase testcases[] = {
```

Comment:
curl uses 2 spaces for indentation.
Applies to the rest of the new unit tests also.